propagator/opentelemetry-propagator-ot-trace/src/opentelemetry/propagators/ot_trace/__init__.py

```diff
@@ -105,7 +105,7 @@ def extract(
 
     def inject(
         self,
-        set_in_carrier: Setter[TextMapPropagatorT],
+        setter: Setter[TextMapPropagatorT],
```

Comment:
should this parameter be optional here? it also appears to be in the wrong order

Reply:
fixed